validator/validator-main.protoascii

```diff
@@ -3145,6 +3145,30 @@ tags: {
     }
   }
 }
+# AMP IMA VIDEO
+tags: {
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-ima-video > script[type=application/json]"
+  mandatory_parent: "AMP-IMA-VIDEO"
+  attrs: {
+    name: "type"
+    mandatory: true
+    value_casei: "application/json"
+  }
+  attrs: {
+    name: "amp-ima-video-settings"
+    mandatory: true
+    value: ""
+    dispatch_key: NAME_DISPATCH
```

Comment:
@Gregable @shawnbuso I think `dispatch_key` can to go `"type"` attr so we don't need the extra `amp-ima-video-settings` attribute.